### DISCUSSION TITLE
Disable 'nicotine' chat room auto-join

### DISCUSSION THREAD

**Discussion by j-minster** [nicotine-plus/nicotine-plus#2519] [General]
Hi, first off thanks for this client. It works quite well. 

I would like to request, however, that the default auto-joining of the 'nicotine' chat room be disabled, or at least let me stop my client from joining it every time I launch. The room is vile and I have no interest in those messages getting passed through my internet connection. What's worse is the messages are logged on disk by default. 

Please, if I've just missed a setting, let me know, but I did dig around for about 10 minutes and found no way to disable the auto-joining of 'nicotine'. 

Thanks.

**Comment by mathiascode** [comment]
See https://github.com/nicotine-plus/nicotine-plus/discussions/2180

I wonder if it would make more sense to simply re-join the rooms from the previous session, similar to private chats. You're not the first person who has overlooked the "Auto-join room" toggle, and it feels somewhat clunky.

**Reply by j-minster** [reply]
Thank you, I never saw that checkbox sorry. The first place I looked was in the settings. I was sure I'd find a list of rooms that would be auto-joined. I normally follow a rule whenever I find a lost item around the house: put it in the first place I looked. It may be worth surveying some people and seeing where they'd first look. 

Cheers, and thanks again for this client. 

Jack.

**Reply by slook** [reply]
> It may be worth surveying some people and seeing where they'd first look.

There should not be any need to look for anything. After the removal of the option check button (in PR #2520) relies simply on open tabs being remembered, closed ones not.

> The room is vile and I have no interest in those messages

It's not nicotine if there's not the guy all capsing racial slurs. Occasionally there has been some interesting tech discussions, but sadly it has become a far departure from whatever the intention of the original developers had for a common meeting area to always be available by default as a center-peice of the application.
